distribution/docs/src/main/jdocs/content/_tables/org.codice.ddf.catalog.plugin.metacard.MetacardIngestNetworkPlugin-table-contents.adoc

```diff
@@ -0,0 +1,35 @@
+:type: documentation
+:status: published
+
+.[[org.codice.ddf.catalog.plugin.metacard.MetacardIngestNetworkPlugin]]Catalog Metacard Ingest Network Plugin
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Criteria
+|criteriaKey
+|String
+|Specifies the criteria for the test of equality; which value will be tested? IP Address? Hostname?
+|remoteAddr
+|true
+
+|Expected Value
+|expectedValue
+|String
+|The value that the criteria must equate to for the attribute overrides to occur.
+|
+|true
+
+|New Attributes
+|newAttributes
+|String"
+|New metacard attributes to apply; if an attribute is specified here, it will be placed on the metacard only if the attribute is not already set. The format should be 'key=value' where 'value' can be a comma-separated list of strings. Whitespace is ignored.
```

Comment:
Ah. Good catch.  We trim both the LHS and RHS after the first regex split on `=`  and we also trim each individual list item on the RHS if it's a list. So white space surrounding the `=` is ignored, and you can't start or end a list item with white space.